test/SlowTests/Client/Counters/SessionCounters.cs

```diff
@@ -1034,5 +1034,247 @@ public void SessionShouldRemoveCountersFromCacheAfterDocumentDeletion()
             }
         }
 
+        [Fact]
+        public void SessionIncludeSingleCounter()
+        {
+            using (var store = GetDocumentStore())
+            {
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new User { Name = "Aviv" }, "users/1-A");
+                    session.CountersFor("users/1-A").Increment("likes", 100);
+                    session.CountersFor("users/1-A").Increment("dislikes", 200);
+
+                    session.SaveChanges();
+                }
+
+                using (var session = store.OpenSession())
+                {
+                    var user = session
+                        .IncludeCounter<User>("likes")
+                        .Load("users/1-A");
+
```

Comment:
What happens if I add `IncludeCounter("dances")` and then try to get it?
It should remember that it is missing

Reply:
this is handled now